src/com/jv_program/lesson1/BinaryOperator.java

```diff
@@ -0,0 +1,6 @@
+package com.jv_program.lesson1;
+
+public interface BinaryOperator< Double> {
```

Comment:
You could use one of the standard function interfaces that appeared in java 8. 
And don't create your own one.